.github/workflows/win-exe.yml

```diff
@@ -188,6 +188,21 @@ jobs:
       semVerStr: ${{ steps.versions.outputs.semVerStr }}
       revNum: ${{ steps.versions.outputs.revNum }}
 
+  publish-winget:
+    name: Publish on winget repo
+    runs-on: windows-latest
+    needs: [build-msi]
+    defaults:
+      run:
+        shell: pwsh
```

Comment:
since this is only one single command, instead of defining a default (that might be unexpected if there ever is the need to add a second step), rather define the shell in the step itself

Reply:
Fixed in [f57d8da](https://github.com/cryptomator/cryptomator/pull/2235/commits/f57d8dab6dc9d57a374f4aadc995a3410422595c)